docs/rule/no-unnecessary-curly-braces-for-strings.md

```diff
@@ -0,0 +1,21 @@
+# no-unnecessary-curly-braces-for-strings
+
+🔧 The `--fix` option on the command line can automatically fix some of the problems reported by this rule.
+
+string values need not be wrapped in the mustache expressions.
+
+## Examples
+
+This rule **forbids** the following:
+
+```hbs
+class={{"btn"}}
```

Comment:
How about a barebones example and/or a more complete example with the entire element:

```suggestion
{{"btn"}}
```

```suggestion
<FooBar class={{"btn"}}>
```